perf.groovy

```diff
@@ -0,0 +1,129 @@
+// Import the utility functionality.
+
+import jobs.generation.*;
+
+// The input project name (e.g. dotnet/corefx)
+def project = GithubProject
+// The input branch name (e.g. master)
+def branch = GithubBranchName
+// Folder that the project jobs reside in (project/branch)
+def projectFolder = Utilities.getFolderName(project) + '/' + Utilities.getFolderName(branch)
+
+// Globals
+
+// Map of os -> osGroup.
+def osGroupMap = ['Ubuntu14.04':'Linux',
```

Comment:
Yeah, this was just boiler plate I got from the netci.groovy that we will need when we eventually add other OSes.